docs/support/tutorials/gis/eo_guide.md

```diff
@@ -0,0 +1,374 @@
+    * Sentinel 2 L1C product
+    * Sentinel 1 SLC, GRD and OCN products
+    * Only Finland (and Baltics)
+
+    [ASF](https://search.asf.alaska.edu/#/)
+    
+    * Needs [registration](https://urs.earthdata.nasa.gov/users/new?)
+    * Sentinel 1 SLC, GRD , RAW and OCN products
+    * Many SAR and SAR derived datasets from other sensors
+    * Worldwide
+    * Sentinel 1 data available for immediate download
+    
+    **All of the above** provide a similar Graphical User Interface (GUI) and Application Programming Interface (API) to access the data.
+    Other tools for downloading the data from open access hubs: [sentinelsat](https://sentinelsat.readthedocs.io/en/stable/) with [examples for SciHub and FinHub](https://github.com/csc-training/geocomputing/blob/master/python/sentinel/sentinelsat_download_from_finhub_and_scihub.py), ...
+
+=== "USGS EarthExplorer"
+
+    [Earthexplorer](https://earthexplorer.usgs.gov/)
+
+    * Needs [registration](https://ers.cr.usgs.gov/register)
+    * Lots of different US related datasets 
+    * Main: Landsat worldwide
+    * GUI in web interface and bulk download
+    * Landsat download instructions: https://lta.cr.usgs.gov/sites/default/files/LS_C2_Help_122020.pdf
+    
+=== "NASA Earthdata"
+
+    [Earthdata](https://search.earthdata.nasa.gov)
+    
+    * Needs [registration](https://urs.earthdata.nasa.gov/users/new)
+    * Harmonized Landsat 8 and Sentinel-2 dataset and many more
+    * Graphical web interface and bulk download
+
+=== "Sentinel image mosaics"
+
+    * Available in Puhti: /appl/data/geo/sentinel
+    * Only Finland
+    * [Sentinel-2 image index mosaics](https://ckan.ymparisto.fi/dataset/sentinel-2-image-index-mosaics-s2ind-sentinel-2-kuvamosaiikit-s2ind) 
+    * [Sentinel-1 SAR-image mosaics](https://ckan.ymparisto.fi/dataset/sentinel-1-sar-image-mosaic-s1sar-sentinel-1-sar-kuvamosaiikki-s1sar)
+    * [WMS (Geoserver)](https://data.nsdc.fmi.fi/geoserver/wms)
+    * [WCS (Geoserver)](https://data.nsdc.fmi.fi/geoserver/wcs)
+    * Provided by [SYKE](https://www.syke.fi/en-US) and [FMI](https://en.ilmatieteenlaitos.fi/)
+
+=== "Google Cloud Storage"
+
+    [Google Cloud Storage](https://cloud.google.com)
+
+    * [Sentinel 2: L1C](https://cloud.google.com/storage/docs/public-datasets/sentinel-2)
+    * [Landsat: Collection 1](https://cloud.google.com/storage/docs/public-datasets/landsat)
+    * [FORCE](../../../apps/force.md) can download directly from here
+
+=== "Amazon Web Service (AWS) *"
+
+
+    * Worldwide
+    * [Sentinel-2 bucket](https://registry.opendata.aws/sentinel-2/)
+    * [Sentinel-1 bucket](https://registry.opendata.aws/sentinel-1/)
+    * Requester pays the download costs
+    * Managed by [Sinergise](http://www.sinergise.com/) 
+
+=== "DIAS *" 
+
+    * Data and Information Access Services
+    * Multiple sites exits:
+        * [ONDA](https://www.onda-dias.eu/cms/)
+        * [sobloo](https://sobloo.eu/)
```

Comment:
Seems like they have shutdown the site? Send people to https://api.oneatlas.airbus.com/guides/extenso/g-extenso-about/ ?